src/Common/tests/Tests/System/Security/IdentityHelperTests.cs

```diff
@@ -36,13 +36,23 @@ public void GetNormalizedStrongNameHash()
         }
 
         [Fact]
-        public void GetNormalizedUrlHash()
+        [PlatformSpecific(TestPlatforms.Windows)]
```

Comment:
Nit: @sepidehMS just went through all of our platform-specific tests and added comments next to them to indicate why they're platform-specific (#15932).  Could you add such comments for these?